lms/static/scripts/frontend_apps/components/EmailNotificationsPreferences.tsx

```diff
@@ -44,63 +77,83 @@ export default function EmailNotificationsPreferences({
   const selectAll = useCallback(() => setAllTo(true), [setAllTo]);
   const selectNone = useCallback(() => setAllTo(false), [setAllTo]);
 
-  return (
-    <Panel
-      onClose={onClose}
-      title="Email Notifications"
-      buttons={<Button variant="primary">Save</Button>}
-    >
-      <p className="font-bold">
-        Receive email notifications when your students annotate.
-      </p>
-      <p className="font-bold">Select the days you{"'"}d like your emails:</p>
+  const onSubmit = useCallback(
+    (event: Event) => onSave({ selectedDays, event }),
+    [onSave, selectedDays]
+  );
 
-      <div className="flex justify-between px-4">
-        <div className="flex flex-col gap-1">
-          {dayNames.map(([day, name]) => (
-            <span
-              key={day}
-              className={classnames(
-                // The checked icon sets fill from the text color
-                'text-grey-6'
-              )}
-            >
-              <Checkbox
-                checked={selectedDays[day]}
-                onChange={() =>
-                  updateSelectedDays({ [day]: !selectedDays[day] })
-                }
-                data-testid={`${day}-checkbox`}
-              >
-                <span
-                  className={classnames(
-                    // Override the color set for the checkbox fill
-                    'text-grey-9'
-                  )}
-                >
-                  {name}
-                </span>
-              </Checkbox>
-            </span>
-          ))}
-        </div>
-        <div className="flex items-start gap-2">
-          <Button
-            variant="secondary"
-            onClick={selectAll}
-            data-testid="select-all-button"
-          >
-            Select all
-          </Button>
+  return (
+    <form onSubmit={onSubmit} method="post" action="">
```

Comment:
Is the empty `action` needed? What is the default if you don't set one?

Reply:
On my initial tests, not setting an action caused the form to not be submit, but there were a couple of things I had to adjust, so perhaps it was something else.

I'll try just removing it, now that it works.